### DISCUSSION TITLE
Application error: a client-side exception has occurred (see the browser console for more information).

### DISCUSSION THREAD

**Discussion by prozaccc** [langgenius/dify#3934] [Help]
The error occurred after running the task with the text generation tool.
<img width="929" alt="iShot_2024-04-28_11 38 02" src="https://github.com/langgenius/dify/assets/26396466/192dae60-6438-4769-af9c-0c8440d70315">

**Comment by zxsimple** [comment]
Exactly the same, when I'm running workflow on the cloud.

**Reply by FableFatale** [reply]
i meet the same question , i closed   page translation  &  translation   extension

**Comment by Upcreat** [comment]
me to

**Comment by patryk20120** [comment]
What is the path and language of the website where error occured?

**Reply by feiyu2000x** [reply]
website： https://udify.app/chatbot/4OEvinl1fL2Hdnhy， Language is Chinese

**Reply by patryk20120** [reply]
Works for me 
![image](https://github.com/langgenius/dify/assets/9155931/33bd7823-faa6-41be-994d-19f31952ef13)

**Reply by feiyu2000x** [reply]
yes,you try to ask the question many times, you can restore the question.Such as, What is a forklift error code? What is TEACHERROR?Curtis1232E-2321控制器，仪表显示故障代码12，怎么解决呢？And Dialog box disappears,can see "Application error:a client-side exception has occurred(see the browser console for more information)."

**Reply by feiyu2000x** [reply]
There will not be an error at the beginning, only one or more questions will report an error.

**Reply by feiyu2000x** [reply]
![Snipaste_2024-05-17_10-04-20](https://github.com/langgenius/dify/assets/108931283/0e35e836-f298-49c9-931a-b0fdde4adcee)

**Reply by feiyu2000x** [reply]
![hh](https://github.com/langgenius/dify/assets/108931283/166ac705-b870-478f-8df8-cd62763c0fd9)
The platform debug chat assistant also has this problem。Could it be a network problem? I use a vpn to access your website

**Comment by feiyu2000x** [comment]
> 刷新就解决了，没必要太在意

不是的，只要提问几个问题窗口就崩溃了，导致用户体验很差，需要刷新才能继续使用。这里面是存在不可忽略的问题的。

**Comment by dualizeo** [comment]
Same Issue, it happens when testing the agent when it ends sending the chat data

**Comment by FableFatale** [comment]
I met the same question. I closed page translation & translation extension

**Reply by Seeumt** [reply]
> I met the same question. I closed page translation & translation extension

Does it work？

**Comment by danialcheung** [comment]
I'm experiencing this issue with versions `0.7.1` and `0.7.2`. I've found that the `Follow-up` feature seems to be the culprit, as disabling the feature seems to fix the issue for now. Don't know if it's an issue on my end or not...

**Comment by daphate** [comment]
I'm getting this happening on the IFRAME inserted page. First time it works fine, after reload - breaks.
Only cleaning of all cookies helps. But that's not the way the actual user will act.

**Reply by Seeumt** [reply]
> I'm getting this happening on the IFRAME inserted page. First time it works fine, after reload - breaks. Only cleaning of all cookies helps. But that's not the way the actual user will act.
Yes， I have experienced the same issue as you several hours ago.
